Makefile

```diff
@@ -1,32 +1,38 @@
-#http://www.gnu.org/prep/standards/html_node/Standard-Targets.html#Standard-Targets
-TOOL_ROOT?=$(shell pwd)/lib/binding
-OSRM_DATASTORE:=$(TOOL_ROOT)/osrm-datastore
-export TOOL_ROOT
-export OSRM_DATASTORE
+# http://www.gnu.org/prep/standards/html_node/Standard-Targets.html#Standard-Targets
 
-all: build
+all: build/Release/osrm.node
 
-pkgconfig:
-	@if [[ `which pkg-config` ]]; then echo "Success: Found pkg-config"; else "echo you need pkg-config installed" && exit 1; fi;
+ifeq (, $(shell which pkg-config))
+ $(error "No pkg-config found: please ensure you have pkg-config installed")
+endif
 
-./node_modules/node-pre-gyp:
+./node_modules/nan:
```

Comment:
Why is this necessary? I was under the impression that `npm install` would get you this.